util/runtime/src/lib.rs

```diff
@@ -54,15 +57,40 @@ impl Handle {
 pub fn new_global_runtime() -> (Handle, StopHandler<()>) {
     let runtime = Builder::new_multi_thread()
         .enable_all()
-        .thread_name("ckb-global-runtime")
+        .thread_name("GlobalRt")
+        .thread_name_fn(|| {
```

Comment:
[The official sample code.](https://docs.rs/tokio/1.7.1/tokio/runtime/struct.Builder.html#method.thread_name_fn)